client/lib/feed-stream-store/feed-stream.js

```diff
@@ -383,7 +383,8 @@ export default class FeedStream {
 					// we don't have the information to render this x-post
 					return false;
 				}
-				const isFollowing = FeedSubscriptionStore.getIsFollowingBySiteUrl( xPostMetadata.siteURL );
+				// TODO: @blowery / @bluefuton -- how can we solve this? Does this solution address the issue
+				const isFollowing = !! find( posts, p => p.site_URL === xPostMetadata.siteURL );
```

Comment:
@bluefuton @blowery: i've done something wacky here -- what can we do better? 
I think the use case for filterXPosts was so that it doesn't appear many times in someone's follows.  Does this solution work (i'll add tests if so)